docs/INSTALLATION.md

```diff
@@ -1,13 +1,15 @@
 ## Dart
-* [Install Dart](https://www.dartlang.org/install)
-* Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* One option is to follow the [instructions for installation](https://www.dartlang.org/install) on dartlang.org.
```

Comment:
Just a minor thing, but instead of ending the sentence with "on dartlang.org" I'd say "on the Dart website", only because there's already a link to the installation page.